### ISSUE TITLE
Taxonomy identifiers on PDB record pages

### ISSUE THREAD

**Issue by cmungall** [biocaddie/prototype_issues#281]
E.g.

https://datamed.org/display-item.php?repository=0002&id=5952cae35152c64c3b0e1a0e&query=neurogenesis

![image](https://user-images.githubusercontent.com/50745/29037314-00025816-7b58-11e7-97c2-2393eccb709d.png)

The ingest seems to be mixing the taxon of the protein of interest with the expression system.

**Comment by ianfore** [comment]
Indeed. There's a loss of information here. The PDB page shows this
Organism: Mus musculus
Expression System: ESCHERICHIA COLI

For the use cases DataMed is intended to address what PDB has as the organism would be the most appropriate i.e. this is a Mouse protein

Yes - information about the expression system is potentially useful but is more an artifact of how the data is obtained. To get into such artifacts for all types of data is beyond a manageable scope for Datamed.
